### PR TITLE
Karma child layouts

### LINKED ISSUE #32: Redesign action pages in karma
Follow this design: https://github.com/AlphaWallet/TokenScript/issues/317

You only need to adapt the existing pages, no need to change XML.

### PR DESCRIPTION
**Pull request by ChintanRathod**
Fixes #32

- Add Market screen layout
- Add Challenge screen layout

### CODE REVIEW

**Comment by ChintanRathod** [comment]
@James-Sangalli This PR has Market and Challenge screen accomplished. This is pending with attestation and will be pushed shortly. Kindly review the PR and share your feedback.

File: examples/Karma/view.en.shtml [hunk 1]
```diff
@@ -107,23 +107,18 @@
...
+						<br />
```
Comment (James-Sangalli) [inline examples/Karma/view.en.shtml:114]: @ChintanRathod this should go in the start.en.shtml page not the view. I will merge this and change it by hand.

**Comment by ChintanRathod** [comment]
@James-Sangalli Thank you for your review. I will push attestation once it is done today.

**Comment by James-Sangalli** [comment]
@ChintanRathod Have made the mods, please pull from master into your fork and then add in attestations

**Pull request merged**